NEWS.md

```diff
@@ -1,5 +1,9 @@
 # purrr 0.2.1.9000
 
+## Bug Fixes
```

Comment:
Please don't add a heading in the NEWS - I do that just prior to release since it's not clear in advance whether or not there will be enough items to warrant subheadings (and what they should be)